pipelines/README.md

```diff
@@ -102,13 +102,41 @@ oc secret link pipeline rhoai-edge-build-pull-secret
 
 ### Deploy and run the test pipeline
 
-> **NOTE**
-> Make sure to change the `target-imagerepo` parameter to match the name of your Quay namespace if using one of the provided `PipelineRun` files.
+Update the `target-imagerepo` parameter value from its default `rhoai-edge` in
+[`test-mlflow-image-pipelinerun-tensorflow-housing.yaml`](tekton/test-mlflow-image-pipeline/test-mlflow-image-pipelinerun-tensorflow-housing.yaml)
+and/or
+[`test-mlflow-image-pipelinerun-bike-rental.yaml`](tekton/test-mlflow-image-pipeline/test-mlflow-image-pipelinerun-bike-rental.yaml)
+to match the name of your Quay namespace.
+
+Then create the pipeline(s) to test the container images behaviour with
+test data in [`tensorflow-housing-test-data-cm.yaml`](tekton/test-mlflow-image-pipeline/tensorflow-housing-test-data-cm.yaml)
+and/or [`bike-rentals-test-data-cm.yaml`](tekton/test-mlflow-image-pipeline/bike-rentals-test-data-cm.yaml):
 
 ```bash
 oc apply -k tekton/test-mlflow-image-pipeline/
 oc create -f tekton/test-mlflow-image-pipeline/test-mlflow-image-pipelinerun-tensorflow-housing.yaml
+# and/or
+oc create -f tekton/test-mlflow-image-pipeline/test-mlflow-image-pipelinerun-bike-rental.yaml
+```
+
+Check what pipeline was created and run and what applications started either in OpenShift Console in Pipelines > Pipelines, Topology > select the application and its Resources and Pod logs, or with CLI find the names of the objects with
```

Comment:
```suggestion
Check what pipeline was created and what applications were started either using:
* OpenShift Console in Pipelines > Pipelines, Topology > select the application and its Resources and Pod logs.

  OR

* CLI; find the names of the objects with:
```

Reply:
This seems to suggest one cannot use both methods.

Reply:
I said OR not XOR 😄 but sure I updated it.